<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTrackedScreenARModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTrackedScreenARModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>CollapsibleButton</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_VideoSource">
        <property name="text">
         <string>Video source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="comboBox_VideoSource">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLVectorVolumeNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_CameraTransform">
        <property name="toolTip">
         <string>This transform will drive the VTK camera in the 3D view.</string>
        </property>
        <property name="text">
         <string>Camera transform:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="comboBox_CameraTransform">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLinearTransformNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_VideoCameraParameters">
        <property name="text">
         <string>Video camera parameters:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="comboBox_VideoCameraParameters">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVideoCameraNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTrackedScreenARModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>comboBox_VideoSource</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>5</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTrackedScreenARModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>comboBox_CameraTransform</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTrackedScreenARModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>comboBox_VideoCameraParameters</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
